### PR TITLE
[#164] restrict volume input parameter

### PR DESCRIPTION
**Pull request by Pacmens**
Restricted inputs to volume to numbers 0-100. added mute message if you set it to 0.

### CODE REVIEW

File: environment/sql/data.sql [hunk 1]
```diff
@@ -71,11 +71,13 @@ INSERT INTO `localisation` (`language`, `name`, `value`) VALUES
 ('en', 'COMMAND_JOIN', 'You need to be in a voice channel to use this command'),
 ('en', 'COMMAND_LOOPQUEUE', 'Loop queue %s'),
 ('en', 'COMMAND_VOLUME', 'Volume set to %d'),
+('en', 'COMMAND_VOLUME_ERROR', 'Volume can only set to a number between 0 and 100'),
```
Comment (AarKro) [inline environment/sql/data.sql]: Should be "Volume can only **be** set to a number between 0 and 100"

File: environment/sql/data.sql [hunk 2]
```diff
@@ -71,11 +71,13 @@ INSERT INTO `localisation` (`language`, `name`, `value`) VALUES
...
+('en', 'COMMAND_VOLUME_MUTE', 'Bot is now muted'),
```
Comment (AarKro) [inline environment/sql/data.sql]: Here I would rather write "Volume is now muted" or only "Volume muted". 
I think in other messages Hera usually refers to her self as "I".

Comment (Pacmens) [reply environment/sql/data.sql]: So "I am now muted" instead since you cant mute a volume

Comment (AarKro) [reply environment/sql/data.sql]: Pretty sure you can say "Volume muted" or the likes, but yeah "I am now muted" is fine with me.

**Pull request merged**